manifest.yml

```diff
@@ -0,0 +1,74 @@
+app-id: com.ilnonop.sonnewebapp
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/run.sh
+finish-args:
+  - --filesystem=home:rw
```

Comment:
why does it need fill home access?

Reply:
Because it has to access the folders ~/.config/SonneWebApp and the folder ~/.local/share/applications

Reply:
`~/.config/SonneWebApp`  is defintely wrong. If it uses the standard directory functions it won't be this in the sandbox, and you don't need any special permissions.